### ISSUE TITLE
Some tables do not open while others open.

### ISSUE THREAD

**Issue by MaximKutovoy** [Trinitek/TpsParser#1]
Some tables do not open or wrong read, while others in base open good. Windows-1251 encoding.

[acc_list.zip](https://github.com/Trinitek/TpsParser/files/3795729/acc_list.zip)
![clarion](https://user-images.githubusercontent.com/57235732/67984798-8f0ef100-fc38-11e9-8a7e-edba72b8b2ab.jpg)

**Comment by MaximKutovoy** [comment]
Please help read this file

**Comment by Trinitek** [comment]
I've identified at least part of the issue, that being that the parser does not interpret `Group` fields correctly.  Your data is being shifted by 10 bytes, which is the total length of the `REU` and `NUM` fields that belong to the `ACC` group.

When I get this fixed and write some tests, would it be okay for me to include your attached file in the integration test?

**Comment by MaximKutovoy** [comment]
Yes, of course. I can send another file where there is a problem.

**Comment by Trinitek** [comment]
The latest commit a1194a9bad3b9609ffb47de12c1bc509640c8f53 should return the correct table structure with a call to `BuildTable()`.  Group members are not expanded, so you will need to iterate them to get the inner values you want.  In other words, the `Value` property on the `ACC` group will be a `IReadOnlyList<TpsObject>`.  The first will be the value for `REU` and the second the value for `NUM`.

I haven't made a new release yet because I'm still working on getting `TpsArray`s to deserialize properly.  If you are interested in using `Deserialize()` instead of calling `BuildTable()`, `TpsGroup` deserialization should work fine for your purposes, but you will need to deserialize the whole group to a `TpsGroup` property and enumerate it the same way as above to get the inner values.

```cs
[TpsField("ACC")]
public TpsGroup Acc { get; set; }
```
```cs
var reu = (string)Acc.Value[0].Value;
var num = (string)Acc.Value[1].Value;
```

**Comment by MaximKutovoy** [comment]
Thank you so much. Now I can get normal data. I will wait for the release.

**Comment by Trinitek** [comment]
Fixed in #5.
